frontend/src/components/pages/logHours.tsx

```diff
@@ -40,6 +41,8 @@ const Feedback = styled.div`
 
 type LogHoursProps = {
   eventData: Event[];
+  setPastShifts: (val: (prev: Shift[]) => Shift[]) => void;
+  setAllShifts: (val: (prev: Shift[]) => Shift[]) => void;
```

Comment:
I added the function `setAllShifts` so an admin will hav an updated `volunteerLog`